logstreams/src/main/java/io/zeebe/logstreams/storage/atomix/AtomixLogStorage.java

```diff
@@ -8,17 +8,22 @@
 package io.zeebe.logstreams.storage.atomix;
 
 import io.atomix.raft.partition.RaftPartition;
-import io.zeebe.logstreams.spi.LogStorage;
-import io.zeebe.logstreams.spi.LogStorageReader;
+import io.zeebe.logstreams.storage.LogStorage;
+import io.zeebe.logstreams.storage.LogStorageReader;
 import java.nio.ByteBuffer;
 import java.util.NoSuchElementException;
 
+/**
+ * Implementation of {@link LogStorage} for the Atomix {@link io.atomix.raft.storage.log.RaftLog}.
+ *
+ * <p>Note that this class cannot be made final because we currently spy on it in our tests. This
+ * should be changed when the log storage implementation is taken out of this module, at which point
+ * it can be made final.
+ */
 public class AtomixLogStorage implements LogStorage {
```

Comment:
Nit: Can we remove/rename Atomix from the name? :smile:

Reply:
What should be call it? RaftLogStorageReader?

Reply:
RaftLogStorage* looks good to me.

Reply:
How about doing this in a follow up issue? I would like to extract the whole package into the broker, and we can rename it then? I don't think it makes sense for the log streams to know about the storage itself. The engine tests for example already have a list storage - we could put that one into the test utils if we ever need to test logstreams in more places.

Reply:
:+1: